apps/web-giddh/src/app/shared/ledger-statement-t-view/utility/ledger-statement.store.ts

```
@@ -0,0 +1,42 @@
+
+import { Injectable, OnDestroy } from "@angular/core";
+import { ComponentStore } from "@ngrx/component-store";
+import { ToasterService } from "../../../services/toaster.service";
+import { select, Store } from "@ngrx/store";
+import { Observable } from "rxjs";
+import { AppState } from "../../../store";
+
+export interface LedgerStatementState {
+}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)